designer/server/src/main/scala/pl/touk/nussknacker/ui/api/RemoteEnvironmentResources.scala

```diff
@@ -56,7 +55,7 @@ class RemoteEnvironmentResources(remoteEnvironment: RemoteEnvironment,
           path(Segment / VersionIdSegment / "migrate") { (processName, version) =>
             (post & processId(processName)) { processId =>
               complete {
-                withProcess(processId.id, version, remoteEnvironment.migrate)
+                withProcess(processId.id, version, remoteEnvironment.migrateScenario)
```

Comment:
That one is just for tests - should we create new endpoint (and new button on UI) for backward compatibility with old migration way?

Reply:
Created flag to use whatever migration we want